src/mixer/basetrackplayer.h

```diff
@@ -100,6 +107,10 @@ class BaseTrackPlayerImpl : public BaseTrackPlayer {
     ControlObjectSlave* m_pPreGain;
     ControlObjectSlave* m_pSpeed;
     ControlObjectSlave* m_pPitchAdjust;
+    QScopedPointer<ControlObjectSlave> m_pInputEnabled;
+    QScopedPointer<ControlObjectSlave> m_pPassthroughEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlStatus;
```

Comment:
Oh .. sorry for the noise, you are right, in the optimized cased all smart pointer produce almost the same code in the
function scope. This is because the whole PIMPL implementation is inlined in the Qt case as well. 
There must have been something with my eyes that I did not notice that in the fist place. 

Other in the class scope, there are is still more code differences even if I set the pointer = NULL in the initializations list. 

We do not need to discuss about how much is much more, since the difference is always only tiny compared to the convenience and safety we have if with smart pointers. 

Please remember, that I have compared a scoped pointer solutions with a plain delete. Both are redundant here because of the QObject tree which offers the same convenience.